src/google-maps/map-marker/map-marker.ts

```diff
@@ -380,64 +391,42 @@ export class MapMarker implements OnInit, OnDestroy, MapAnchorPoint {
     return this.marker;
   }
 
-  private _combineOptions(): Observable<google.maps.MarkerOptions> {
-    return combineLatest([this._options, this._title, this._position, this._label, this._clickable])
-        .pipe(map(([options, title, position, label, clickable]) => {
-          const combinedOptions: google.maps.MarkerOptions = {
-            ...options,
-            title: title || options.title,
-            position: position || options.position,
-            label: label || options.label,
-            clickable: clickable !== undefined ? clickable : options.clickable,
-            map: this._googleMap.googleMap,
-          };
-          return combinedOptions;
-        }));
+  /** Creates a combined options object using the passed-in options and the individual inputs. */
+  private _combineOptions(): google.maps.MarkerOptions {
+    const options = this._options || DEFAULT_MARKER_OPTIONS;
+    return {
+      ...options,
+      title: this._title || options.title,
+      position: this._position || options.position,
+      label: this._label || options.label,
+      clickable: this._clickable !== undefined ? this._clickable : options.clickable,
+      map: this._googleMap.googleMap,
+    };
   }
 
-  private _watchForOptionsChanges() {
-    this._options.pipe(takeUntil(this._destroy)).subscribe(options => {
-      if (this.marker) {
-        this._assertInitialized();
-        this.marker.setOptions(options);
-      }
-    });
-  }
+  /** Syncs the input values with the marker object. */
+  private _syncInputs() {
+    const {marker, _title, _position, _label, _clickable} = this;
 
-  private _watchForTitleChanges() {
-    this._title.pipe(takeUntil(this._destroy)).subscribe(title => {
-      if (this.marker && title !== undefined) {
-        this._assertInitialized();
-        this.marker.setTitle(title);
+    if (marker) {
+      marker.setOptions(this._combineOptions());
+
+      if (_title !== undefined) {
```

Comment:
Since we're setting combined options on line 412, isn't this redundant?

Reply:
Same for other individual setters below.

Reply:
Actually, see other comment. I think there is an issue more generally with this method of doing it.

Reply:
Which issue are you referring to in your last comment?

Reply:
Sorry, wasn't quite done typing it yet, it's there now.